<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TnDMainWin</class>
 <widget class="QWidget" name="TnDMainWin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>820</width>
    <height>531</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TnDMainWin</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(67, 67, 67);
font: 9pt &quot;Microsoft YaHei UI&quot;;</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>GroupBox</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QPushButton" name="pushButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(141, 94, 0);
background-color: rgb(60, 121, 182);</string>
            </property>
            <property name="text">
             <string> Today's Transfers </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_2">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(141, 94, 0);
background-color: rgb(60, 121, 182);</string>
            </property>
            <property name="text">
             <string>Delivery</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_3">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(141, 94, 0);
background-color: rgb(60, 121, 182);</string>
            </property>
            <property name="text">
             <string>Start Day</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_4">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(141, 94, 0);
background-color: rgb(60, 121, 182);</string>
            </property>
            <property name="text">
             <string>End Day</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::MinimumExpanding</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_5">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(141, 94, 0);
</string>
            </property>
            <property name="text">
             <string>Additional Transfers</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QWidget" name="widget" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QTabWidget" name="tabWidget">
            <property name="tabPosition">
             <enum>QTabWidget::West</enum>
            </property>
            <property name="tabShape">
             <enum>QTabWidget::Rounded</enum>
            </property>
            <property name="currentIndex">
             <number>0</number>
            </property>
            <property name="iconSize">
             <size>
              <width>32</width>
              <height>32</height>
             </size>
            </property>
            <property name="elideMode">
             <enum>Qt::ElideLeft</enum>
            </property>
            <property name="documentMode">
             <bool>true</bool>
            </property>
            <widget class="QWidget" name="tab">
             <attribute name="title">
              <string>Transfers</string>
             </attribute>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <layout class="QFormLayout" name="formLayout">
                <property name="fieldGrowthPolicy">
                 <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
                </property>
                <item row="0" column="0">
                 <widget class="QLabel" name="label">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="text">
                   <string>Item Type</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="QComboBox" name="comboBox">
                  <property name="styleSheet">
                   <string notr="true">background-color: rgb(141, 94, 0);
background-color: rgb(60, 121, 182);</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0">
                 <widget class="QLabel" name="label_2">
                  <property name="text">
                   <string>Item Name</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1">
                 <widget class="QComboBox" name="comboBox_2">
                  <property name="styleSheet">
                   <string notr="true">background-color: rgb(141, 94, 0);
background-color: rgb(60, 121, 182);</string>
                  </property>
                 </widget>
                </item>
                <item row="3" column="0">
                 <widget class="QLineEdit" name="lineEdit">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="readOnly">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item row="4" column="0">
                 <widget class="QLabel" name="label_3">
                  <property name="text">
                   <string>Transferred to :</string>
                  </property>
                 </widget>
                </item>
                <item row="4" column="1">
                 <widget class="QComboBox" name="comboBox_3">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">background-color: rgb(141, 94, 0);
background-color: rgb(60, 121, 182);</string>
                  </property>
                  <item>
                   <property name="text">
                    <string>JKIA</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>PEPONI - BAR</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>PEPONI - KITCHEN</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>UN CAMPUS</string>
                   </property>
                  </item>
                 </widget>
                </item>
                <item row="6" column="0">
                 <widget class="QLabel" name="label_4">
                  <property name="text">
                   <string>Units : </string>
                  </property>
                 </widget>
                </item>
                <item row="6" column="1">
                 <widget class="QLineEdit" name="lineEdit_2">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="minimumSize">
                   <size>
                    <width>450</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>450</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <property name="readOnly">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item row="7" column="0">
                 <widget class="QLabel" name="label_5">
                  <property name="text">
                   <string>Quantity Transferred</string>
                  </property>
                 </widget>
                </item>
                <item row="7" column="1">
                 <widget class="QLineEdit" name="lineEdit_4"/>
                </item>
                <item row="8" column="0">
                 <widget class="QLabel" name="label_6">
                  <property name="text">
                   <string>Issued By </string>
                  </property>
                 </widget>
                </item>
                <item row="8" column="1">
                 <widget class="QLineEdit" name="lineEdit_3"/>
                </item>
                <item row="9" column="1">
                 <spacer name="verticalSpacer_2">
                  <property name="orientation">
                   <enum>Qt::Vertical</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>20</width>
                    <height>40</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item row="10" column="1">
                 <widget class="QPushButton" name="pushButton_6">
                  <property name="styleSheet">
                   <string notr="true">background-color: rgb(141, 94, 0);
background-color: rgb(60, 121, 182);</string>
                  </property>
                  <property name="text">
                   <string>Issue Transfer</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="0">
                 <widget class="QPushButton" name="pushButton_7">
                  <property name="styleSheet">
                   <string notr="true">background-color: rgb(145, 145, 145);</string>
                  </property>
                  <property name="text">
                   <string>Refresh</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="tab_2">
             <attribute name="title">
              <string>Deliveries</string>
             </attribute>
            </widget>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
